bin/rialto/node/src/chain_spec.rs

```diff
@@ -187,6 +187,6 @@ fn load_kovan_bridge_config() -> Option<BridgeKovanConfig> {
 fn load_millau_bridge_config() -> Option<BridgeMillauConfig> {
 	Some(BridgeMillauConfig {
 		init_data: None,
-		owner: Some([0; 32].into()),
+		owner: None,
```

Comment:
Who's going to initialize it then?

Reply:
Root can initialize

Reply:
But the relayer is supposed to call this, no? I mean does it currently go through `sudo` module? Why not set `root_key` as the owner here as well (which would be `//Alice`).

Since this is a testnet/development network I feel this would be much easier to use also from the UI if the calls don't have to be `sudo`-wrapped.